stdlib/2and3/optparse.pyi

```diff
@@ -187,8 +187,8 @@ class OptionParser(OptionContainer):
     def _get_args(self, args: Iterable) -> List[Any]: ...
     def _init_parsing_state(self) -> None: ...
     def _match_long_opt(self, opt: _Text) -> _Text: ...
-    def _populate_option_list(self, option_list: List[Option], add_help: bool = ...) -> None: ...
-    def _process_args(self, largs: List[_Text], rargs: List, values: Values) -> None: ...
+    def _populate_option_list(self, option_list: Sequence[Option], add_help: bool = ...) -> None: ...
+    def _process_args(self, largs: Sequence[_Text], rargs: List, values: Values) -> None: ...
```

Comment:
Since it's just an internal method I'm changing it back to plain `List` (implying `List[Any]`) -- that's what's used for `rargs` too, and they are pretty similar -- both must be lists (or MutableMappings?) of "strings" (per documentation).